src/sentry/search/events/fields.py

```diff
@@ -392,9 +391,6 @@ def normalize_count_if_value(args: Mapping[str, str]) -> Union[float, str, int]:
     # TODO: not supporting field aliases or arrays yet
     elif column in FIELD_ALIASES or column in ARRAY_FIELDS:
         raise InvalidSearchQuery(f"{column} is not supported by count_if")
-    # At this point only string or tag columns are left
-    elif condition not in ["equals", "notEquals"]:
```

Comment:
Should this be expanded to cover every condition instead? Or I guess rather, if they pass an invalid condition how do we return an error without this code?

Reply:
Conditions are already validated elsewhere
https://github.com/getsentry/sentry/blob/bd521f8478b41d1698c00ae8e075877bf517af93/src/sentry/search/events/fields.py#L1079-L1088